locked-asset/proxy-dex/src/proxy_common.rs

```diff
@@ -4,14 +4,8 @@ elrond_wasm::derive_imports!();
 use common_structs::Nonce;
 use common_structs::{WrappedFarmTokenAttributes, WrappedLpTokenAttributes};
 
-pub const ACCEPT_PAY_FUNC_NAME: &[u8] = b"acceptPay";
```

Comment:
Is the proxy contract a payable contract now? How can I tell if a contract is payable?

Reply:
Yes. No idea